src/styles/globals.css

```diff
@@ -13,15 +13,33 @@
   :root {
     --background: 0 0% 100%;
     --foreground: 222.2 84% 4.9%;
+    
 
     --card: 0 0% 100%;
     --card-foreground: 222.2 84% 4.9%;
 
     --popover: 0 0% 100%;
     --popover-foreground: 222.2 84% 4.9%;
-
-    --primary: 222.2 47.4% 11.2%;
-    --primary-foreground: 210 40% 98%;
+    
+    --primary-50: 260, 100%, 99%;
+    --primary-100: 260, 100%, 98%;
+    --primary-200: 261, 53%, 94%;
+    --primary-300: 261, 48%, 90%;
+    --primary-400: 262, 53%, 79%;
+    --primary-500: 263, 64%, 65%;
+    --primary-600: 262, 50%, 49%;
+    --primary-700: 262, 49%, 33%;
+    --primary-800: 262, 31%, 20%;
+
+    --primary: var(--primary-800);
+    --primary-foreground: var(--primary-100);
+
+    --violet-100: 257, 58%, 95%;
+    --violet-200: 250, 33%, 91%;
+    --violet-300: 248, 28%, 79%;
+    --violet-400: 247, 30%, 70%;
+    --violet-500: 247, 14%, 52%;
+    --violet-600: 247, 15%, 34%;
```

Comment:
yo, ur gonna have to resolve conflicts here. I added the colours already

Reply:
yeah I just copy pasted urs in, so it shouldn't conflict

Reply:
github literally says there a conflict tho ://// it also wouldn't be showing in the diff here if the conflicts were removed.

Reply:
<img width="1002" alt="image" src="https://github.com/user-attachments/assets/65b9a0ff-b73e-4cc7-847f-587d79bbcf3e">